app/controllers/new_invite_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class NewInviteController < ApplicationController
```

Comment:
An empty controller?

Reply:
I am reviewing it as well and just checked. When the controller is removed then hard reload throws an error

<img width="541" alt="Screenshot 2024-10-29 at 11 35 22 am" src="https://github.com/user-attachments/assets/ece82cb1-9313-4830-b7a8-cc9470fcfd49">

Reply:
Yeah the controller exists so that Rails can figure out how to route requests that land on `/new-invite`.